commands/info/emoji.js

```diff
@@ -7,7 +7,7 @@ const RichEmbed = require('../../structures/RichEmbed');
 class EmojiInfoCommand extends Command {
 	constructor(handler) {
 		super(handler, {
-			aliases: ['emoji'],
+			aliases: ['einfo', 'ee'],
```

Comment:
Just checked... I thought you named the `emoji.js` command as emoji, not emojiinfo. Most commands we put the name of it as the file name.

Reply:
We do.
You moved the `emojiinfo.js` from `commands/common` to `commands/info` and renamed the file.
I am not too sure about renaming the class to `EmojiCommand` to match the file name again, since that feels a bit too broad.
Would be consistent though, so probably a yes.
Both `emoji` and `emojiinfo` should call this command regardless.

Reply:
I see, my bad... I renamed it because... A lot of end users do not check the aliases, and emojiinfo is... a bit long @.@